opentelemetry-jaeger/src/lib.rs

```diff
@@ -144,6 +144,12 @@
 //! }
 //! ```
 //!
+//! ## Crate Feature Flags
+//!
+//! The following crate feature flags are available:
+//!
+//! * `collector_client`: Export span data directly to a Jaeger collector.
```

Comment:
Could be helpful to mention the underlying http client
```suggestion
//! * `collector_client`: Export span data directly to a Jaeger collector using isahc http client.
```